python/cuspatial/cuspatial/_lib/interpolate.pyx

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from libcpp.memory cimport make_unique
+from cudf._libxx.table cimport *
```

Comment:
We're trying to move away from importing wildcards to explicitly importing what's needed